packages/polymorphic/src/Polymorphic.spec.tsx

```diff
@@ -0,0 +1,505 @@
+import React from 'react';
+import { parseTSDoc } from '../../../scripts/utils/tsDocParser';
+import { render } from '@testing-library/react';
+import styled from '@emotion/styled';
+import { Polymorph, usePolymorphicRef, type PolymorphicComponentType } from '.';
+import {
+  ExamplePolymorphic,
+  ExamplePolymorphicWithRef,
+  ExampleInferred,
+  AdvancedPolymorphic,
+  AdvancedPolymorphicWithRef,
+  RestrictedExample,
+} from './Example';
+import { makeWrapperComponent } from './utils/Polymorphic.testutils';
+
+describe('packages/polymorphic', () => {
```

Comment:
should we break this into two files: testing polymorph and polymorphic?

Reply:
Probably!